asterisk_logcall/models/asterisk_server.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Phone Log-call module for Odoo/OpenERP
+#    Copyright (C) 2016 credativ Ltd (<http://credativ.co.uk>).
+#    Copyright (C) 2016 Trever L. Adams
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, api
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+
+class AsteriskServer(models.Model):
+    _inherit = "asterisk.server"
+
+    ucp_url = fields.Char(
+        string='Script to download FreeSWITCH recordings', required=False,
+        default="https://localhost/cgi-bin/get_recording.pl?file={odoo_filename}",
+        help="Macros allowed: {odoo_type} (inbound, outbound), {odoo_src}"
+        "(source phone number}, {odoo_dst} (destination number), "
+        "{odoo_duration} (length of call), {odoo_start} (start time of call "
+        "in seconds since epoch), {odoo_filename} (file name on server), "
+        "{odoo_uniqueid} (FreeSWITCH UUID of call).")
+    server_jitter_correction = fields.Integer(
+        string='Time jitter compensation', required=True,
+        help='Number of seconds to subtract from new call start and add to '
```

Comment:
On 08/25/2016 05:45 PM, Eric Caudal wrote:
>
> In asterisk_logcall/models/asterisk_server.py
> <https://github.com/OCA/connector-telephony/pull/101#discussion_r76345099>:
>
> > +
> > +
> > +class AsteriskServer(models.Model):
> > +    _inherit = "asterisk.server"
> > +
> > +    ucp_url = fields.Char(
> > +        string='Script to download FreeSWITCH recordings', required=False,
> > +        default="https://localhost/cgi-bin/get_recording.pl?file={odoo_filename}",
> > +        help="Macros allowed: {odoo_type} (inbound, outbound), {odoo_src}"
> > +        "(source phone number}, {odoo_dst} (destination number), "
> > +        "{odoo_duration} (length of call), {odoo_start} (start time of call "
> > +        "in seconds since epoch), {odoo_filename} (file name on server), "
> > +        "{odoo_uniqueid} (FreeSWITCH UUID of call).")
> > +    server_jitter_correction = fields.Integer(
> > +        string='Time jitter compensation', required=True,
> > +        help='Number of seconds to subtract from new call start and add to '
>
> Thanks for the help messages.
> Would it make sense to gather them in the README? (nice to have)
>
Should I remove them from the help messages? I have put them into the
README in an appropriate format.

Trever